python/lsst/afw/table/source/source.cc

```diff
@@ -261,4 +245,4 @@ PYBIND11_PLUGIN(_source) {
     return mod.ptr();
 }
 
-}}}  // namespace lsst::afw::table
+}}}}  // namespace lsst::afw::table::<anonymous>
```

Comment:
Why is the module in the anonymous namespace? Only the declare functions should be.